api/src/test/java/org/pmiops/workbench/billing/BillingProjectBufferServiceTest.java

```diff
@@ -529,6 +544,55 @@ public void cleanBillingBuffer_ignoreValidEntries() {
         .isEqualTo(ASSIGNING);
   }
 
+  @Test
+  public void cleanBillingBuffer_multipleCreates() {
+    billingProjectBufferService.bufferBillingProjects();
+
+    CLOCK.setInstant(NOW.plus(30, ChronoUnit.MINUTES));
+    billingProjectBufferService.bufferBillingProjects();
+    billingProjectBufferService.bufferBillingProjects();
+
+    final ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
+    verify(fireCloudService, times(3)).createAllOfUsBillingProject(captor.capture());
+    final List<String> capturedProjectNames = captor.getAllValues();
+    doReturn(new BillingProjectStatus().creationStatus(CreationStatusEnum.CREATING))
```

Comment:
The need to have a mocked return paired with each input may indicate that we'd benefit from a stand-along test fixture that could take in a map (for example). If there were such a fixture for the billing buffer, I might have a small use for it in the billing status PR test.

Reply:
An in-memory fake for the Firecloud API would probably be useful.

Reply:
Let's start making a list of such things... It'd be good to settle on some fake design principles potentially.